WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -167,7 +173,20 @@ final class HubMenuViewModel: ObservableObject {
         navigationPath.append(HubMenuNavigationDestination.payments)
     }
 
-    private func setupPOSElement() {
+    func showReviewDetails(using parcel: ProductReviewFromNoteParcel) {
+        productReviewFromNoteParcel = parcel
+        showingReviewDetail = true
+    }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self, name: .setUpTapToPayViewDidAppear, object: nil)
+    }
+}
+
+// MARK: - Helper methods
+//
+private extension HubMenuViewModel {
```

Comment:
nit: We could move the method `createCardPresentPaymentService` inside this private extension.

Reply:
Done in 5c000c4a32d994c7d56c7d0ce14562a58243bdff.